### DISCUSSION TITLE
新增次数兑换

### DISCUSSION THREAD

**Discussion by lmuiotctf** [lobehub/lobehub#126] [General]
### 🥰 需求描述 | Feature Description

需要新增一个token次数的兑换需求

### 🧐 解决方案 | Proposed Solution

新增一个弹窗兑换入口，可以实现兑换码进行兑换

**Comment by arvinxx** [comment]
Lobe Chat 作为一个开源 Chatbot，暂不提供 token 充值相关的服务能力。 LobeChat  目前的定位暂时只是一个可以供 DIY 能力强的用户自行部署的三方OpenAI 客户端，当然未来也会支持 Claude 、开源 LLM 等。

你提到的兑换诉求，我们未来会在 LobeHub 的 Web 服务中提供。

**Comment by arvinxx** [comment]
目前想使用的话，可以使用类似 https://api.chatanywhere.cn 这样的服务来获取 Api Key。

PS：该服务不是我们提供的，只是我自己有在用，比官方 API 要便宜一些。
